common/buildcraft/factory/block/TankBlockConnector.java

```diff
@@ -0,0 +1,8 @@
+package buildcraft.factory.block;
+
+import buildcraft.factory.tile.TileTank;
+
+/** Marker interface for {@link BlockTank} to determine if the block below should be considered a tank block when
+ * visually connecting below. This should only really be implemented when the block creates an instance of
+ * {@link TileTank}. */
+public interface TankBlockConnector {}
```

Comment:
woudn't it be better if we add an ITank interface to the api and use that instead?

Reply:
It's specific to Buildcraft internals - a tile entity must extend TileTank for it to work, so there's little point in moving it to the api (provided for irontanks)